src/Helper/Placeholder/Container/AbstractStandalone.php

```diff
@@ -176,6 +176,24 @@ protected function escape($string)
         return $this->getEscaper()->escapeHtml((string) $string);
     }
 
+    /**
+     * Escape an attribute value
+     *
+     * @param  string $string
+     * @return string
+     */
+    protected function escapeAttribute($string)
```

Comment:
Add the return type and parameter type, please. They can be dropped from the docblock

Reply:
But don't we still support PHP 5.6?
In the `composer.json` there is `"php": "^5.6 || ^7.0"`

Reply:
URRRRGGGHHHHHHHHHHH @weierophinney any note on that?

Reply:
Until we bump to a new major version, we support PHP 5.6.

Reply:
Do we leave it like that or...?

Reply:
What? Since when is a new major needed for version bumps? That's horrible :-\

Reply:
@Ocramius I've told you this more than once! It was on our blog post announcing the move to 7.1 (https://framework.zend.com/blog/2017-06-06-zf-php-7-1.html):

> - We will start releasing new major versions of components that pin to PHP 7.1+, so we can take advantage of the latest language features.
> - We will no longer test against HHVM. Most of our components were failing against HHVM already, so this does not pose a BC break at this time.
> - We will continue providing security patches only to the last release branch supporting PHP 5.6 until PHP 5.6 reaches its end-of-life date. This allows us to minimize the amount of maintenance we need for supporting these versions, while still providing stable, secure releases for our PHP 5 users.
> - New components will pin to PHP 7.1+.

Only new components, or new major versions of components, are supposed to bump to 7.1. The idea is to allow us to refactor to provide all necessary typehints when we do, which will break BC. Adding typehints is a BC break, thus new major version.

Reply:
So anything that was in `zendframework/zendframework` is not free to bump, while anything outside of it can bump at each minor?

Reply:
Note: thinking about `zendframework/zend-code` (for example), which can only really support the latest PHP minor each time

Reply:
@weierophinney If I understand correctly, bumping the minimum PHP version from 5.6 to 7.x is a BC (the major version of the repository must also be changed). However, if we bump the minimum PHP version from 7.1 to 7.2, it's a minor change, so we would only increment the minor version of the repository. Right?

Reply:
Correct. We have dropped PHP 5.5 support in minor versions throughout the 2.Y series of releases on all components.

Reply:
Reminder that PHP-SRC does not follow SemVer

Reply:
Water under the bridge at this point. We adopted this policy because it provides a consistent, understandable message: we will work with new major versions of PHP, but will not adopt their features until we issue a new major release.

When folks see a new 2.Y release come out, but Composer says it cannot update to it, and all bugfixes and security fixes are in that release and forwards, we receive a lot of pushback, as users cannot then update to receive these fixes. Many have then requested having multiple minor release branches for purposes of releasing security fixes — which becomes an even greater maintenance nightmare than we currently have. With a new major release, there is plenty of messaging of changes, users have to opt-in to the new version, and users know they need to update by the end of the support period if they want security fixes.

(For the record, we receive pushback even for bumping a minor PHP version within minor versions! However, it's generally easier to update to a new minor PHP release than to a new major one.)

Reply:
Is there anything else that we can do for this PR?

Reply:
@thexpand I think PR is fine, sorry for derailing the convo

Reply:
@Ocramius Actually, I'm glad that such a discussion happened :)

Reply:
Side note - is there anything that we can do for the failing Travis build with lowest deps on PHP 7.2? It seems to be happening to almost all PRs of `zend-view`.